tests/conftest.py

```diff
@@ -2274,6 +2275,20 @@ def cleanup(self):
     on_exit.cleanup()
 
 
+@pytest.fixture(scope="session", autouse=True)
+def add_mgmt_test_mark(duthosts):
+    '''
+    @summary: Create mark file at /etc/sonic/mgmt_test_mark, and DUT can use this mark to detect mgmt test.
+    @param duthosts: fixture to get DUT hosts
+    '''
+    mark_file = "/etc/sonic/mgmt_test_mark"
+    for duthost in duthosts:
+        try:
+            duthost.shell("touch %s" % mark_file, module_ignore_errors=True)
```

Comment:
You can use `duthosts.shell` as well. It will automatically dispatch the shell module to each duthost.
With `module_ignore_error` should be enough. No need to catch other exceptions. When the DUT is unreachable, the exception will be handled by other functions.

Reply:
Updated.